pkg/testutil/leak.go

```diff
@@ -106,6 +106,7 @@ func interestingGoroutines() (gs []string) {
 		}
 		stack := strings.TrimSpace(sl[1])
 		if stack == "" ||
+			strings.Contains(stack, "runtime/sigqueue.go") ||
```

Comment:
is there a more descriptive string than sigqueue.go? like the name of the goroutine that's hanging?